### PR TITLE
TRE-308 Update documentation with V3 api and new rules regarding charge amount

### PR DESCRIPTION
**Pull request by ameliemaucade**
Not ready to be merged, missing : 

- [x] note for merchants for full flex campaigns 
- [ ] screenshots for the campaigns

See preview of the changes : 
campaigns_1
campaign_2
list_agreements
charge_amount

### CODE REVIEW

**Comment by cloveras** [comment]
Something for you, @rognlien?

I merged the Swagger updates in https://github.com/vippsas/vipps-recurring-api/pull/172

**Review by rebeksburnett — COMMENTED** [review]
The changes looked good, from an editorial perspective. I can't assess the technical content yet. (-: I saw a few things I wanted to fix, but these were already there and I didn't want to delay this PR. I did see one typo: miliseconds -> milliseconds.    
Is it better to push a fix to the PR for something small like this or to make a comment?

File: vipps-recurring-api.md [hunk 1]
```diff
@@ -590,14 +681,26 @@ This is an example response from a call to
...
+### List agreements
+
+It is possible to retrieve all agreements by calling [`GET:/recurring/v3/agreements`](https://vippsas.github.io/vipps-recurring-api/#/Agreement%20V3%20Endpoints/ListAgreementsV3)
+By default, this endpoint will return only "ACTIVE" agreements.
+
+Query parameters available 
+| Query parameter               | Description                                 | Example                                     |
+| ----------------------------- | ------------------------------------------- | ------------------------------------------- |
+| `status`                      | Filter by status of the agreement(PENDING, ACTIVE, EXPIRED, STOPPED) | ?status=ACTIVE     |
+| `createdAfter`                | Filter by agreement start date in miliseconds | ?createdAfter=1645543020311               |
```
Comment (rebeksburnett) [inline vipps-recurring-api.md:700]: miliseconds -> milliseconds

File: vipps-recurring-api.md [hunk 2]
```diff
@@ -637,9 +740,13 @@ For example, in the agreement
 a limit of 2495 NOK (499 x 5) would be in place. If this limit becomes a
 hindrance the agreement `price` can be [updated](#update-an-agreement).
 
-**Note:** Although it is _technically_ possible to increase the price 10
-times, we **strongly** recommend to be as user-friendly as possible, and
-to make sure the user understands any changes and get updated information.
+From **August 1st, 2022** new rules will be enforced :
```
Comment (rebeksburnett) [inline vipps-recurring-api.md:743]: Shouldn't have a space before the ":". Not a biggie though. I can fix it later.

File: vipps-recurring-api.md [hunk 3]
```diff
@@ -536,36 +559,104 @@ when the product is shipped.
 
 ### Campaigns
 
-A campaign in recurring is a period where the price is lower than usual, and
-this is communicated to the customer with the original price shown for comparison.
+Campaigns can be used to lower the price of the agreement for a limited amount of time.
+Recurring supports 4 different campaign types : price campaign, period campaign, event campaign, full flex campaign, see more about the different campaign types in the table below. This is communicated to the customer with the original price shown for comparison.
```
Comment (rebeksburnett) [inline vipps-recurring-api.md:563]: " :"  should not have the space before.

Comment (rebeksburnett) [reply vipps-recurring-api.md:563]: I would have said: Recurring supports 4 different campaign types: price campaign, period campaign, event campaign, and full flex campaign. See more about the different campaign types in the table below.

File: vipps-recurring-api.md (hunk 1 above)
Comment (ameliemaucade) [reply vipps-recurring-api.md]: Thanks, fixed

File: vipps-recurring-api.md (hunk 2 above)
Comment (ameliemaucade) [reply vipps-recurring-api.md]: Good catch, fixed it everywhere else in the file as well. 
It's a habit I have from French 🇫🇷 , we put a space before the ":" 😄

File: vipps-recurring-api.md (hunk 3 above)
Comment (ameliemaucade) [reply vipps-recurring-api.md]: Done 👍

File: vipps-recurring-api.md [hunk 4]
```diff
@@ -560,7 +568,7 @@ when the product is shipped.
 ### Campaigns
 
 Campaigns can be used to lower the price of the agreement for a limited amount of time.
-Recurring supports 4 different campaign types: price campaign, period campaign, event campaign, and full flex campaign. See more about the different campaign types in the table below. This is communicated to the customer with the original price shown for comparison.
```
Comment (rebeksburnett) [inline vipps-recurring-api.md]: No biggie, but we lost the space here, after the ":".

File: vipps-recurring-api.md [hunk 5]
```diff
@@ -537,36 +566,110 @@ when the product is shipped.
...
+Recurring supports 4 different campaign types: price campaign, period campaign, event campaign, and full flex campaign. See more about the different campaign types in the table below. This is communicated to the customer with the original price shown for comparison.
 Campaigns can not be used in combination with variable amount, see more about variable amount [here](#Recurring-agreements-with-variable-amount).
 
-![Campaign example](images/CampaignExample.PNG)
+| Campaign types        | Description                                                                                              | Example                                                                                 |
+|-----------------------|----------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------|
+| `price campaign`      | A set price until a set date. Same interval as agreement.                                                | 10kr per 1 week interval until 2022-05-01T00:00:00Z and then 200kr per 1 week interval  |
```
Comment (skasti) [inline vipps-recurring-api.md:575]:
```suggestion
| `price campaign`      | Different interval price until specified date. Same interval as agreement.                                                | 10kr per 1 week interval until 2022-05-01T00:00:00Z and then 200kr per 1 week interval  |
```

A lot of people will have trouble understanding the difference between "A set price until a set date" and "A set price for a given duration". I suggest being a bit more specific so they might understand without needing the example.

File: vipps-recurring-api.md [hunk 6]
```diff
@@ -537,36 +566,110 @@ when the product is shipped.
...
+| `period campaign`     | A set price for a given duration. A duration is defined by a number of periods (DAY, WEEK, MONTH, YEAR)  | 1kr for 6 months and then 100kr each month                                              |
+| `event campaign`      | A set price for until a given date with a text description for a given event                             | 20kr until Christmas and then 100kr each month                                          |
```
Comment (skasti) [inline vipps-recurring-api.md:577]:
```suggestion
| `event campaign`      | A set price until a given event date with a text describing the event                             | 20kr until Christmas and then 100kr each month                                          |
```

File: vipps-recurring-api.md [hunk 7]
```diff
@@ -537,36 +566,110 @@ when the product is shipped.
...
+| `full flex campaign`  | A set price for a different interval until a given date                                                  | 10kr per week until 2022-05-01T00:00:00Z and then 200kr a month                         |
```
Comment (skasti) [inline vipps-recurring-api.md:578]:
```suggestion
| `full flex campaign`  | Different price and interval until a given date                                                  | 10kr per week until 2022-05-01T00:00:00Z and then 200kr a month                         |
```

File: vipps-recurring-api.md [hunk 8]
```diff
@@ -537,36 +566,110 @@ when the product is shipped.
...
+    }
   }
 }
 ```
+| Field          | Description                                                                    |
+|----------------|--------------------------------------------------------------------------------|
+| `campaignType` | The type of the campaign                                                       |
+| `price`        | The price that the customer will pay for the period of the campaign            |
+| `period.unit`  | The period where the campaign price is applied. Can be DAY, WEEK, MONTH, YEAR  |
+| `period.count` | The number of periods the campaign should run for                              |
```
Comment (skasti) [inline vipps-recurring-api.md:620]:
```suggestion
| `period`  | The period where the campaign price is applied. Consist of a Unit and a Count, example; { "unit": "WEEK", "count": 4 }  |
```

I think the explanation is very clunky. Period is a data structure used elsewhere and shouldn't need re-explanation everywhere

**Comment by ameliemaucade** [comment]
Closing PR. Not relevant anymore.

**Pull request closed without merging**